tests/unit/test_scriptHandler.py

```diff
@@ -29,7 +29,8 @@ def script_test(self, gesture):
 
 		self.assertEqual(script_test.__doc__, "description")
 		self.assertEqual(script_test.category, SCRCAT_MISC)
-		self.assertItemsEqual(script_test.gestures, ["kb:a", "kb:b", "kb:c"])
+		# #9720 (Py3 review required): self.assertItemsEqual -> self.assertCountEqual.

```

Comment:
Again as above, I don't think this should be in the eventual code of this pr if we just take the rename for granted.